example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,19 +1,34 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.example">
-    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
You might need to add back the internet permission in the example.

Reply:
The latest commit should outdate this file, I wonder why GitHub doesn't mark it accordingly...

Reply:
Everything looks good from my side I can see that the permission has been added 👍